types/rebass__layout/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for @rebass/layout 4.0
+// Project: https://github.com/rebassjs/rebass#readme
+// Definitions by: Rafael Almeida <https://github.com/rafaelalmeidatk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import * as Rebass from 'rebass';
+
+export {};
+
```

Comment:
```suggestion
```
this is for special cases (mark as a module, mark as module to prevent exporting top level internal types). Should be OK without that one.

Reply:
Oh cool, I was unsure it was really needed, thanks!